<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>579</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File manager with ML</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>901</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab1">
          <attribute name="title">
           <string>View Folders</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>621</width>
             <height>511</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Selected Folder Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="placeholderText">
               <string>Search for specific folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listView2"/>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab2">
          <attribute name="title">
           <string>Select Folder</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>139</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="selectFolderButton">
              <property name="text">
               <string>Select Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="start_Button">
              <property name="text">
               <string>Start</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop_Button">
              <property name="text">
               <string>Stop</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="finish_Button">
              <property name="text">
               <string>Finish</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>5</x>
             <y>150</y>
             <width>201</width>
             <height>271</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Folder Statistics</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>212</height>
               </size>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Selected Folder Path</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrev_Directory"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Algorithms</string>
    </property>
    <widget class="QMenu" name="menuSupervised">
     <property name="title">
      <string>Supervised</string>
     </property>
     <addaction name="actionChoose_an_algorithm_to_train"/>
    </widget>
    <widget class="QMenu" name="menuUnsupervised">
     <property name="title">
      <string>Unsupervised</string>
     </property>
     <addaction name="actionchoose_an_algorithm_to_train"/>
    </widget>
    <addaction name="menuSupervised"/>
    <addaction name="menuUnsupervised"/>
    <addaction name="separator"/>
    <addaction name="actionAll_Files"/>
    <addaction name="actionDocuments"/>
    <addaction name="actionImages"/>
    <addaction name="actionMusic"/>
    <addaction name="actionVideos"/>
    <addaction name="separator"/>
    <addaction name="actionProgramming"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRowColour"/>
    <addaction name="actionStatistics"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionKeyboard_Shortcuts"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New Process</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent Process</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionKeyboard_Shortcuts">
   <property name="text">
    <string>Keyboard Shortcuts</string>
   </property>
  </action>
  <action name="actionRowColour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Alternating Row Colours</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAll_Files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Files</string>
   </property>
  </action>
  <action name="actionDocuments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Documents</string>
   </property>
  </action>
  <action name="actionImages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="actionVideos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Videos</string>
   </property>
  </action>
  <action name="actionMusic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </action>
  <action name="actionChoose_an_algorithm_to_train">
   <property name="text">
    <string>Choose an algorithm to train</string>
   </property>
  </action>
  <action name="actionchoose_an_algorithm_to_train">
   <property name="text">
    <string>Choose an algorithm to train</string>
   </property>
  </action>
  <action name="actionPrev_Directory">
   <property name="text">
    <string>Prev-Directory</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionProgramming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Programming</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons_resources_file.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
